ui/plugin-system/src/components/LegendOptionsEditor/LegendOptionsEditor.tsx

```diff
@@ -140,9 +161,25 @@ export function LegendOptionsEditor({ value, onChange }: LegendOptionsEditorProp
           ></SettingsAutocomplete>
         }
       />
+      <OptionsEditorControl
+        label="Size"
+        control={
+          <SettingsAutocomplete
+            value={{
+              ...legendSizeConfig,
+              id: currentSize,
+            }}
+            options={SIZE_OPTIONS}
+            onChange={handleLegendSizeChange}
+            // TODO: enable sizes for list mode when we normalize the layout of
+            // lists to more closely match tables.
+            disabled={value === undefined || currentMode !== 'Table'}
+            disableClearable
+          ></SettingsAutocomplete>
+        }
+      />
       <OptionsEditorControl
         label="Values"
-        description="Computed values ignore nulls."
```

Comment:
I forgot to take this note out when I updated the calculations. It's no longer valid, so ripping it out now.